framework/test/jest/config/setup.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * LiskHQ/lisk-core2
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+require('jest-extended');
+require('jest-chain');
```

Comment:
Can you explain why we need `jest-chain` and `jest-extended` ?

Reply:
`jest-extended` provides you some nice matchers.

`jest-chain` https://github.com/mattphillips/jest-chain#problem

Reply:
Please, add those cases to the any example, otherwise you propose them without any reason.

Reply:
I am adding unit tests in https://github.com/LiskHQ/lisk/issues/2839